staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/openapi_aggregator.go

```diff
@@ -0,0 +1,314 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
nit: you can get rid of the openapi_ prefixes in the file name now.

Reply:
Good idea.